### ISSUE TITLE
Get better scores

### ISSUE THREAD

**Issue by turner11** [turner11/VQA-MED#39]
**2019 first result:**

| Loss function | activation | trainable parameters | BLEU | WBSS| DB ID |
| ------------- | ------------- | ------------- | ------------- | ------------- |------------- |
cosine_proximity | sigmoid | 120220 | 0.5003 | 0.4922 | 5 |5 |
cosine_proximity | sigmoid | 120220 | 0.4925 | 0.4879 | 11 |11 |
cosine_proximity | sigmoid | 120220 | 0.4890 | 0.4834 | 14 |14 |
cosine_proximity | sigmoid | 120220 | 0.4821 | 0.4786 | 12 |12 |
cosine_proximity | tanh | 120220 | 0.4489 | 0.4486 | 4 |4 |
cosine_proximity | tanh | 120220 | 0.4442 | 0.4430 | 20 |20 |
cosine_proximity | tanh | 120220 | 0.4399 | 0.4404 | 32 |32 |
cosine_proximity | tanh | 120220 | 0.4383 | 0.4398 | 26 |26 |
cosine_proximity | tanh | 120220 | 0.4393 | 0.4356 | 38 |38 |
cosine_proximity | tanh | 127207 | 0.4303 | 0.3032 | 21 |21 |

**2018 top results:**

| Loss function | activation | trainable parameters | BLEU | WBSS| DB ID |
| ------------- | ------------- | ------------- | ------------- | ------------- |------------- |
mean_absolute_percentage_error | relu | 147389 | 0.1594 | 0.3628 | 126 |"
mean_absolute_percentage_error | relu | 83901 | 0.1594 | 0.3628 | 121 |"
mean_absolute_error | relu | 147389 | 0.1525 | 0.3511 | 70 |"
mean_absolute_percentage_error | relu | 83901 | 0.1526 | 0.3494 | 125 |"
cosine_proximity | tanh | 83901 | 0.2132 | 0.1698 | 85 |"
cosine_proximity | sigmoid | 147389 | 0.2210 | 0.1693 | 77 |"
cosine_proximity | relu | 147389 | 0.2212 | 0.1679 | 98 |"
poisson | softmax | 147389 | 0.2151 | 0.1617 | 107 |"
